packages/markdown-docx/src/OOXMLTransformer/index.js

```diff
@@ -58,171 +87,178 @@ class OoxmlTransformer {
     }
 
     /**
-     * Gets the value of the variable
+     * Constructs a ciceroMark Node JSON from the information
      *
-     * @param {Array} elements the variable elements
-     * @returns {string} the value of the variable
+     * @param {object} nodeInformation Contains properties and value of a node
+     * @return {object} CiceroMark Node
      */
-    getValue(elements) {
-        const variableContents = elements[1];
-        for (const property of variableContents.elements[0].elements) {
-            if (property.name === 'w:t') {
-                return property.elements[0].text;
-            }
+    construtCiceroMarkNodeJSON(nodeInformation) {
```

Comment:
`construtCiceroMarkNodeJSON` and `constructNodes` purposes are not clear from just reading the names of the methods or even docstrings. You have even mentioned `or block CiceroMark Node` in docstrings of `constructNodes` so why was `construtCiceroMarkNodeJSON` necessary in the first place?

Perusing your code made a little clearer that former creates atomic nodes like text, softbreak, etc. While the latter creates blocks for heading paragraph. So just write their method names properly **without** relying too much on the docstrings.

Reply:
The spelling of "construct" is wrong. I have suggested this before and I will suggest again using an IDE or a VSCode plugin for spell checkers.

Reply:
> `construtCiceroMarkNodeJSON` and `constructNodes` purposes are not clear from just reading the names of the methods or even docstrings. You have even mentioned `or block CiceroMark Node` in docstrings of `constructNodes` so why was `construtCiceroMarkNodeJSON` necessary in the first place?
> 
> Perusing your code made a little clearer that former creates atomic nodes like text, softbreak, etc. While the latter creates blocks for heading paragraph. So just write their method names properly **without** relying too much on the docstrings.

`construtCiceroMarkNodeJSON`: It creates a text, softbreak or any other inline node
`constructNodes`:This creates all the nodes present in ap paragraph, or heading node.

This it the exact use of the above.